Dockerfile.test

```diff
@@ -16,18 +18,21 @@ RUN wget http://python.org/ftp/python/$pyVer/Python-${pyVer}.tgz && \
 	make && \
 	checkinstall
 
-RUN apt-get install -y python-dev python-pip
-RUN pip install psutil && \
-    pip install netifaces && \
-	pip install bottle && \
-	pip install requests && \
-	pip install python-dateutil
-
 RUN mkdir /crawler
 COPY . /crawler/
 WORKDIR /crawler/
 
-RUN pip2 install --no-index --find-links="packaging/python_packages/" psutil bottle requests pydoubles netifaces kafka-python docker-py pykafka
+RUN apt-get install -y python-dev python-pip
```

Comment:
We just need python 2.7

On Tue, Jun 14, 2016 at 7:41 PM, Morgan Bauer <notifications@github.com>
wrote:

> In Dockerfile.test
> <https://github.com/cloudviz/agentless-system-crawler/pull/26#discussion_r67077692>
> :
>
> >  RUN mkdir /crawler
> >  COPY . /crawler/
> >  WORKDIR /crawler/
> >
> > -RUN pip2 install --no-index --find-links="packaging/python_packages/" psutil bottle requests pydoubles netifaces kafka-python docker-py pykafka
> > +RUN apt-get install -y python-dev python-pip
>
> is python 2.7.5 such a hard dependency that we can't just install python
> with apt-get?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cloudviz/agentless-system-crawler/pull/26/files/bf1dd26f113d76b412db149b21b9b8d635e2b983#r67077692>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK6ydRzCkm5lHmYi22PFuoGiVM8RUeFwks5qLzwPgaJpZM4I07Zq>
> .
>